packages/salesforcedx-vscode-core/src/commands/forceOrgOpen.ts

```diff
@@ -6,24 +6,90 @@
  */
 
 import {
+  CliCommandExecutor,
   Command,
   SfdxCommandBuilder
 } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as vscode from 'vscode';
+import { channelService } from '../channels';
 import { nls } from '../messages';
+import { telemetryService } from '../telemetry';
+import { getRootWorkspacePath } from '../util';
 import {
   EmptyParametersGatherer,
   SfdxCommandlet,
   SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from './commands';
 
+interface CLIOrgData {
+  status: number;
+  result: {
+    orgId: string;
+    url: string;
+    username: string;
+  };
+}
+
 class ForceOrgOpenExecutor extends SfdxCommandletExecutor<{}> {
   public build(data: {}): Command {
-    return new SfdxCommandBuilder()
+    const builder = new SfdxCommandBuilder()
       .withDescription(nls.localize('force_org_open_default_scratch_org_text'))
       .withArg('force:org:open')
-      .withLogName('force_org_open_default_scratch_org')
-      .build();
+      .withLogName('force_org_open_default_scratch_org');
+    if (process.env.SFDX_CONTAINER_MODE) {
+      builder.withArg('--urlonly').withJson();
+    }
+    return builder.build();
+  }
+
+  private buildUserMessageWith(orgData: CLIOrgData): string {
+    return nls.localize(
+      'force_org_open_container_mode_message_text',
+      orgData.result.orgId,
+      orgData.result.username,
+      orgData.result.url
+    );
+  }
```

Comment:
This method will produce a formatted message based on the JSON response from the CLI to match the message displayed  to the user when they run the command without the `--json` flag.

```
Access org 00D1U000000qfglUAA as user jhork@dev.com with the following URL: https://na85.salesforce.com/secur/frontdoor.jsp?sid=00D1U000000qfgl!AQ4AQAD_1.NwP.soL74jcx5MlM0nXLeBiCxJmn626qsyLXXJK6byVDKseNB8aLSx.VKSHQaiwf5XjUd7p7e1f6hmEjaL5Q.Z
```